Unit test failure - api.tests.ConvertFileUploadTests converting XLS spreadsheet

The following is the output from the unit tests.  This seems to have been failing since the last 2 upgrades of the SPDX Tools jar file.

```
.....11:40:19.804 [Thread-4] ERROR org.spdx.spreadsheetstore.SpdxSpreadsheet - I/O error reading SPDX Spreadsheet
java.io.IOException: Your InputStream was neither an OLE2 stream, nor an OOXML stream
        at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:258) ~[tool.jar:?]
        at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:221) ~[tool.jar:?]
        at org.spdx.spreadsheetstore.SpdxSpreadsheet.<init>(SpdxSpreadsheet.java:107) ~[tool.jar:?]
        at org.spdx.spreadsheetstore.SpreadsheetStore.deSerialize(SpreadsheetStore.java:329) ~[tool.jar:?]
        at org.spdx.tools.SpdxConverter.convert(SpdxConverter.java:148) ~[tool.jar:?]
F11:40:19.881 [Thread-5] ERROR org.spdx.spreadsheetstore.SpdxSpreadsheet - I/O error reading SPDX Spreadsheet
java.io.IOException: Your InputStream was neither an OLE2 stream, nor an OOXML stream
        at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:258) ~[tool.jar:?]
        at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:221) ~[tool.jar:?]
        at org.spdx.spreadsheetstore.SpdxSpreadsheet.<init>(SpdxSpreadsheet.java:107) ~[tool.jar:?]
        at org.spdx.spreadsheetstore.SpreadsheetStore.deSerialize(SpreadsheetStore.java:329) ~[tool.jar:?]
        at org.spdx.tools.SpdxConverter.convert(SpdxConverter.java:148) ~[tool.jar:?]
F
======================================================================
FAIL: test_convert_xlsxtordf_api (api.tests.ConvertFileUploadTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\gary\git\spdx-online-tools\src\api\tests.py", line 178, in test_convert_xlsxtordf_api
    self.assertTrue(resp.status_code==406 or resp.status_code == 201)
AssertionError: False is not true

======================================================================
FAIL: test_convert_xlsxtotag_api (api.tests.ConvertFileUploadTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\gary\git\spdx-online-tools\src\api\tests.py", line 186, in test_convert_xlsxtotag_api
    self.assertTrue(resp.status_code==406 or resp.status_code == 201)
AssertionError: False is not true

----------------------------------------------------------------------
```

--- a/src/api/tests.py
+++ b/src/api/tests.py
@@ -32,7 +32,7 @@ from json import dumps, loads
 from api.oauth import generate_github_access_token,get_user_from_token
 from api.views import generateLicenseXml
 
-from api.models import ValidateFileUpload,ConvertFileUpload,CompareFileUpload,CheckLicenseFileUpload,SubmitLicenseModel
+from api.models import ValidateFileUpload,ConvertFileUpload,CompareFileUpload,SubmitLicenseModel
 from django.conf import settings
 import os
 
@@ -290,7 +290,6 @@ class CheckLicenseFileUploadTests(APITestCase):
             u.delete()
         except ObjectDoesNotExist:
             pass
-        CheckLicenseFileUpload.objects.all().delete()
 
     def test_checklicense_api(self):
         """Access get without login"""

